homeassistant/components/zha/entities/entity.py

```diff
@@ -0,0 +1,89 @@
+"""
+Entity for Zigbee Home Automation.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/zha/
+"""
+from homeassistant.helpers import entity
+from homeassistant.util import slugify
+from homeassistant.core import callback
+
+
+class Entity(entity.Entity):
```

Comment:
Let's call this `ZhaEntity`to distinguish it from the base home assistant `Entity` class.

Reply:
Done. Thanks for the quick review!